packages/design-system/src/styles/components/_Tabs.scss

```diff
@@ -26,16 +26,16 @@
 .ds-c-tabs__item,
 .ds-c-tabs__item:visited {
   appearance: none;
-  background-color: $color-background;
-  border-bottom: 1px solid $border-color;
-  border-left: 1px solid $border-color;
+  background-color: $tabs__background-color;
+  border-bottom: 1px solid $tabs__border-color;
+  border-left: 1px solid $tabs__border-color;
   border-radius: 0;
-  border-top: 1px solid $border-color;
-  color: $color-base;
+  border-top: 1px solid $tabs__border-color;
+  color: $tabs__color;
   display: inline-block;
   font-size: $font-size-sm;
-  font-weight: $font-bold;
-  line-height: $reset-line-height;
+  font-weight: $font-weight-bold;
+  line-height: $font-line-height-reset;
```

Comment:
This will probably be revisited with later work, but should font styles like these be defined under the "tabs" theme?

Reply:
I'm focusing on colors primarily right now as a pilot, eventually will add more if we decide to make these more customizeable here.

Reply:
So these are just changes to reference the new token name values.